packages/webapp/src/components/PolicyAndCookieBanner/PolicyAndCookieBanner.types.ts

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+export enum CookieConsentCategories {
+	/**
+	 * Cookies to perform essential website functions (sign-in, language settings,...)
+	 */
+	Required = 'Required',
+	/**
+	 * Cookies to understand how website is used, may be also used on 3rd party websites that are not owned or operated by Microsoft
+	 */
+	Analytics = 'Analytics',
+	/**
+	 * Cookies to show ads and content based on user social media profiles and activity on Microsoft websites
+	 */
+	SocialMedia = 'SocialMedia',
+	/**
+	 * Cookies to record which ads are already seen, clicked, or purchased
+	 */
+	Advertising = 'Advertising',
+}
+
+export type CookieConsentBannerThemes = 'light' | 'dark' | 'high-contrast'
+
+export type CookieConsent = Record<CookieConsentCategories, boolean>
+
+export type CookieConsentManager = {
+	/**
+	 * `true` if consent is required for current user region
+	 */
+	readonly isConsentRequired: boolean
+
+	/**
+	 * Returns consent state for all categories
+	 */
+	getConsent(): CookieConsent
+
+	/**
+	 * Returns consent state for a category.
+	 * @param consentCategory one of `consentCategories` values to get the consent state for
+	 * @returns `true` if consent was given, `false` otherwise
+	 */
+	getConsentFor(consentCategory: CookieConsentCategories): boolean
+
+	/**
+	 * Shows the preferences dialog box
+	 */
+	manageConsent(): void
+}
+
+/* eslint-disable-next-line */
+export type WcpConsent = {
+	/**
+	 * Library initilization method
```

Comment:
I think there is a small typo in here: `initi_lization` -> `initialization` ->

Reply:
Thanks!